beginner_source/nlp/word_embeddings_tutorial.py

```diff
@@ -208,10 +208,15 @@
 And see thy blood warm when thou feel'st it cold.""".split()
 # we should tokenize the input, but we will ignore that for now
 # build a list of tuples.  Each tuple is ([ word_i-2, word_i-1 ], target word)
-trigrams = [([test_sentence[i], test_sentence[i + 1]], test_sentence[i + 2])
-            for i in range(len(test_sentence) - 2)]
-# print the first 3, just so you can see what they look like
-print(trigrams[:3])
+ngrams = [
```

Comment:
The comment above would need to be updated no?

Reply:
yes